4.6.2-windowsservercore-ltsc2016/runtime/Dockerfile

```diff
@@ -9,10 +9,9 @@ RUN Add-WindowsFeature Web-Server; `
     Add-WindowsFeature Web-Asp-Net45; `
     Remove-Item -Recurse C:\inetpub\wwwroot\*
 
-COPY ServiceMonitor.exe /
-
 #download Roslyn nupkg and ngen the compiler binaries
```

Comment:
This comment is a little misleading now the ServiceMonitor is also grouped in.  Logically, it feels like adding the ServiceMonitor better belongs with the previous RUN instruction.  Thoughts?

Reply:
I can fix the comment, but I believe the overarching question is should I minimize the number of layers or optimize for optimize logically grouping contents of a layer. My preference is to minimize number of layers. I'd like to hear your thoughts?

Reply:
I agree with minimizing the layers.  That being said, I was trying to point out that it feels to me the ServiceMonitor logic better belongs in the web-server setup layer (e.g. the previous RUN instruction).  The current proposal is not consistent in the pattern used across the various Dockerfiles.  Is this intentional and if so why?

Reply:
Not intentional. Just a mistake